src/explorer/pagerankTable/TableRow.js

```diff
@@ -95,3 +106,12 @@ export function PaddingRow(props: {|+backgroundColor: string|}) {
     </tr>
   );
 }
+
+const styles = StyleSheet.create({
+  hoverHighlight: {
+    backgroundSize: "100% 100%, 0 0",
+    ":hover, :focus": {
```

Comment:
This style is applying to the `:focus` pseudo-selector for _every_
element on the page: <https://github.com/Khan/aphrodite/issues/196>.
You can tell because the focus-styling for `<button>` and `<select>`
elements has changed.

You can get the desired _code_ by using two pseudo-selectors…

```diff
diff --git a/src/explorer/pagerankTable/TableRow.js b/src/explorer/pagerankTable/TableRow.js
index 762f80f4..0f38d9df 100644
--- a/src/explorer/pagerankTable/TableRow.js
+++ b/src/explorer/pagerankTable/TableRow.js
@@ -110,7 +110,10 @@ export function PaddingRow(props: {|+backgroundColor: string|}) {
 const styles = StyleSheet.create({
   hoverHighlight: {
     backgroundSize: "100% 100%, 0 0",
-    ":hover, :focus": {
+    ":hover": {
+      backgroundSize: "100% 100%, 100% 100%",
+    },
+    ":focus": {
       backgroundSize: "100% 100%, 100% 100%",
     },
   },
```

…but after testing this I see that it’s not actually achieving the
desired _effect_, because `<tr>`s don’t take focus, so this selector
never applies.

Try it: as you tab through the page, you focus on the various buttons
and links, but the row never gains highlight.

The solution is to use [`:focus-within`][fw] instead of `:focus`.

[fw]: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within